reconfiguration/cmf/concord.cmf

```diff
@@ -94,6 +94,11 @@ Msg AddRemoveStatusResponse 28 {
     string reconfiguration
 }
 
+Msg AddRemoveWithWedgeCommand 29 {
+    uint64 sender_id
+    string reconfiguration
```

Comment:
I would change the name to "config_descriptor" (as this is what it is going to be)

Reply:
Also, I would add a bft filed (like in the wedge command), this way we give the operator the control if to wait for n/n wedge or n-f wedge

Reply:
fixed